.eslintrc.json

```diff
@@ -48,7 +48,10 @@
     "class-methods-use-this": "off",
     "no-debugger": "off",
     "import/order": "off",
-    "no-console": "off"
+    "no-console": "off",
+    "no-param-reassign": "off",
+    "import/no-extraneous-dependencies": "off",
+    "no-constant-condition": "off"
```

Comment:
Dodałam to, ponieważ kilka razy w algorytmie jest while(true), ale nie mam pomysłu jak to wyeliminować.